cypress/integration/my-orcid-sidebar.e2e.js

```diff
@@ -0,0 +1,118 @@
+describe('My Orcid sidebar' + runInfo(), () => {
+  describe('Orcid id', () => {
+    it('display the users Orcid', () => {})
+    it('display url to navigate to the public page view', () => {})
+  })
+  describe('Emails' + runInfo(), () => {
+    it('display a user with only a primary email "unverified"', () => {})
+    it('display a user with only a primary email "verified"', () => {})
+    it('display multiple emails "unverified" and "verified"', () => {
+      // Expect changes to be display outside and inside of the modal
+    })
+    describe('Emails edit modals' + runInfo(), () => {
+      it('add additional emails and display those as "unverified" with default privacy', () => {
+        // Expect changes to be display outside and inside of the modal
+      })
+      it('remove emails', () => {
+        // Expect the primary email to not display the delete button
+        // Expect changes to be display outside and inside of the modal
+      })
+      it('edit a non-primary and display it as "unverified"', () => {
+        // Expect changes to be display outside and inside of the modal
+      })
+      it('edit a primary email and display it as "verified"', () => {
```

Comment:
Should we prompt a "please verify your primary email" type of message after a primary email is edited? as the current app does

Reply:
As concluded on previous meetings, edit the primary email won't be allow